indexing-service/src/main/java/io/druid/indexing/overlord/ForkingTaskRunner.java

```diff
@@ -388,16 +760,191 @@ public void shutdown(final String taskid)
           @Override
           public InputStream openStream() throws IOException
           {
-            return LogUtils.streamFile(processHolder.logFile, offset);
+            return LogUtils.streamFile(logFile, offset);
           }
         }
     );
   }
 
+  private void populateMissingTasksFromDir()
```

Comment:
Part of the annoyance here is that `listFiles` can return null instead of an empty array. How sucky is that?